src/modules/Providers/utils/components/DetailsPage/DetailItem.tsx

```diff
@@ -166,6 +95,32 @@ const DescriptionTitle: FC<{ title: string }> = ({ title }) => (
   <DescriptionListTerm> {title} </DescriptionListTerm>
 );
 
+/**
+ * Component for displaying an item's title
+ *
+ * @component
+ */
+const DisplayTitle: FC<{
+  title: string;
+  helpContent: ReactNode;
+  showHelpIconNextToTitle: boolean;
+  moreInfoLabel?: string;
```

Comment:
Probably worth pulling this into its own type definition called DisplayTitleProps

Reply:
I wasn't really refactoring this component, I had one minor change in this file, which caused linters to fail on used before defined rule, this component is out of my refactoring scope as it lies under Providers (should be moved to src/components)